logbook-spring/src/test/java/org/zalando/logbook/spring/BufferingClientHttpResponseWrapperTest.java

```diff
@@ -0,0 +1,82 @@
+package org.zalando.logbook.spring;
```

Comment:
It would be nice to have a more realistic test which verifies that a stream can be read multiple times.

Reply:
Do you have a suggestion for a more realistic test? The Spring Mock Server always returns a byte array input stream which doesn't exhibit the original problem.

Reply:
Ok, I've adapted the test to do real network requests instead of mock requests. This way, if the `BufferingClientHttpResponseWrapper` is not used, the original "mark/reset not supported" exception will be thrown.

Reply:
> This way, if the `BufferingClientHttpResponseWrapper` is not used, the original "mark/reset not supported" exception will be thrown.

Perfect! Exactly the kind of test I like for this kind problem.